external/fv3fit/fv3fit/keras/_models/precipitative.py

```diff
@@ -35,27 +35,21 @@
 Q_TENDENCY_NAME = "dQ2"
 
 
-def integrate_precip(args):
-    dQ2, delp = args  # layers seem to take in a single argument
-    # output should be kg/m^2/s
-    return tf.math.scalar_mul(
-        # dQ2 * delp = s-1 * Pa = s^-1 * kg m-1 s-2 = kg m-1 s-3
-        # dQ2 * delp / g = kg m-1 s-3 / (m s-2) = kg/m^2/s
-        tf.constant(-1.0 / GRAVITY, dtype=dQ2.dtype),
-        tf.math.reduce_sum(tf.math.multiply(dQ2, delp), axis=-1),
-    )
-
+class IntegratePrecipLayer(tf.keras.layers.Layer):
+    def call(self, args) -> tf.Tensor:
+        dQ2, delp = args  # layers seem to take in a single argument
```

Comment:
What was the error when using multiple args?  I've used layers before with specific arguments for each tensor and I think it worked.

https://github.com/ai2cm/fv3net/blob/d4924b40618a363a5ad27a98b6fb410f541d7a48/external/fv3fit/fv3fit/emulation/layers/fields.py#L89

Reply:
Descriptions are embedded in comments here.  Make consistent with `CondensationalHeatingLayer`

Reply:
There wasn't an error, I just (back when I wrote the original code here, the comment is pre-existing) noticed keras seems to have a "style" where most layers take in a list of inputs, instead of separate arguments.

Thanks for catching the dropped docstrings, added docstrings to both layers.